libbpf-tools/slabratetop.h

```diff
@@ -0,0 +1,13 @@
+/* SPDX-License-Identifier: (LGPL-2.1 OR BSD-2-Clause) */
+#ifndef __SLABRATETOP_H
+#define __SLABRATETOP_H
+
+#define CACHE_NAME_SIZE 32
+
+struct slabrate_info {
+    char name[CACHE_NAME_SIZE];
```

Comment:
And here.

Reply:
> And here.

Fix https://github.com/iovisor/bcc/pull/3797/commits/bdd1c1265cca653917e173b6047c3935524fb6c0

Reply:
> And here.

Thank you for your corrections